mjpc/planners/ilqg/backward_pass.cc

```diff
@@ -111,14 +113,36 @@ int iLQGBackwardPass::RiccatiStep(
   mju_addTo(Quut, cuut, m * m);
 
   //----- regularize ----- //
-  mju_copy(Qxu_reg, Qxut, n * m);
-  mju_copy(Quu_reg, Quut, m * m);
+  if (reg_type == kCostToGoRegularization) {
```

Comment:
I prefer kValueRegularization (and elsewhere)

Reply:
Done.